libs/ett/core/shell/src/lib/ett-shell.component.ts

```diff
@@ -16,14 +16,26 @@
  */
 import {
   ChangeDetectionStrategy,
+  ChangeDetectorRef,
   Component,
   NgModule,
+  OnInit,
   ViewEncapsulation,
 } from '@angular/core';
-import { RouterModule } from '@angular/router';
+import { Router, RouterModule } from '@angular/router';
+import {
+  AuthOidcHttp,
+  GetProfileResponse,
+  UserProfile,
+} from '@energinet-datahub/ett/auth/data-access-api';
 import { WattShellModule } from '@energinet-datahub/watt';
+import { WattButtonModule } from '@energinet-datahub/watt';
+import { MatIconModule } from '@angular/material/icon'; // TODO Import from Watt?
+import { MatMenuModule } from '@angular/material/menu'; // TODO Import from Watt?
+import { map, catchError } from 'rxjs';
```

Comment:
issue: remove unused import
```suggestion
import { map, } from 'rxjs';
```